### PR TITLE
Issue login redirect

### PR DESCRIPTION
**Pull request by JonnyTischbein**
To issue #5818 I fixed former unset `$_SESSION['return_url']` so if you access a site without logged in you will be redirected to this page after a successful login.
If a login failed you are redirected to `/login` and after successful login here to `/network`.

I added the login mask in many pages (like it was did in `/admin` and `/network`). Did I miss pages ?

For more information see https://github.com/friendica/friendica/issues/5818#issuecomment-427462884

### CODE REVIEW

**Comment by JonnyTischbein** [comment]
My branch root is messed up in /mod/contats.php

**Comment by MrPetovan** [comment]
What do you mean?

File: mod/contacts.php [hunk 1]
```diff
@@ -635,15 +636,15 @@ function contacts_content(App $a, $update = 0)
 			'$follow_text' => $follow_text,
 			'$profile_select' => $profile_select,
 			'$contact_id' => $contact['id'],
-			'$block_text' => ($contact['blocked'] ? L10n::t('Unblock') : L10n::t('Block')),
-			'$ignore_text' => ($contact['readonly'] ? L10n::t('Unignore') : L10n::t('Ignore')),
-			'$insecure' => (in_array($contact['network'], [Protocol::ACTIVITYPUB, Protocol::DFRN, Protocol::MAIL, Protocol::DIASPORA]) ? '' : $insecure),
+			'$block_text' => (($contact['blocked']) ? L10n::t('Unblock') : L10n::t('Block') ),
```
Comment (JonnyTischbein) [inline mod/contacts.php]: I didn't do these changes.

Comment (MrPetovan) [reply mod/contacts.php]: Indeed, you're reverting recent changes by Michael. This means your `issue_login_redirect` needs to be rebased onto `develop`. Here's how in your Friendica folder: 
```
git fetch --all
git rebase friendica/develop issue_login_redirect
git push --force
```

Comment (JonnyTischbein) [reply mod/contacts.php]: after rebase it just sad everything is uptodate.. I keep trying around

**Pull request closed without merging**